OpenDreamClient/EntryPoint.cs

```diff
@@ -77,12 +77,6 @@ public sealed class EntryPoint : GameClient {
 
             _dreamInterface.Initialize();
             IoCManager.Resolve<IDreamSoundEngine>().Initialize();
-
-            if (_configurationManager.GetCVar(CVars.DisplayCompat))
-                _dreamInterface.OpenAlert(
-                    "Compatibility Mode Warning",
-                    "You are using compatibility mode. Clicking in-game objects is not supported in this mode.",
-                    "Ok", null, null, null);
```

Comment:
Clicking still doesn't work in compatibility mode. This alert needs brought back, along with avoiding calling `GetPixel()` now that it's a crash instead of skipped over.